mne/report.py

```diff
@@ -1919,19 +1921,25 @@ def _render_epochs(self, epo_fname, image_format):
             caption=caption, show=show, image_format=image_format)
         return html
 
-    def _render_cov(self, cov_fname, info_fname, image_format):
+    def _render_cov(self, cov_fname, info_fname, image_format, show_svd=True):
         """Render cov."""
         global_id = self._get_id()
         cov = read_cov(cov_fname)
-        fig, _ = plot_cov(cov, info_fname, show=False)
-        img = _fig_to_img(fig, image_format)
-        caption = 'Covariance : %s (n_samples: %s)' % (cov_fname, cov.nfree)
-        show = True
-        html = image_template.substitute(
-            img=img, id=global_id, div_klass='covariance',
-            img_klass='covariance', caption=caption, show=show,
-            image_format=image_format)
-        return html
+        fig, svd = plot_cov(cov, info_fname, show=False, show_svd=show_svd)
+        html = []
+        figs = [fig]
+        captions = ['Covariance : %s (n_samples: %s)' % (cov_fname, cov.nfree)]
+        if svd is not None:
+            figs.append(svd)
+            captions.append('Singular values of the noise covariance')
+        for fig, caption in zip(figs, captions):
+            img = _fig_to_img(fig, image_format)
+            show = True
+            html.append(image_template.substitute(
+                img=img, id=global_id, div_klass='covariance',
+                img_klass='covariance', caption=caption, show=show,
+                image_format=image_format))
+        return '\n'.join(html)
```

Comment:
This change captures both the covariance figure and the SVD figure, and iterate over them to add both to the report if they are both present. Previously, the SVD figure was automatically created, and ended up inline in the tutorial (rather than getting captured and put into the report HTML).  Currently the choice of whether to show/not show the SVD plot is internal (e.g., user can't choose when instantiating `mne.Report` or when calling `report.parse_folder`).  LMK if you think it should be exposed (and if so, where to expose it and under what name).

Reply:
I think it should always be shown in the report